lib/src/definitions.dart

```diff
@@ -64,18 +64,23 @@ class MapViewConfiguration {
   /// Default is false.
   final bool enableDebugging;
 
+  ///Set if you want to lock the camera to the building and stop the user from moving away
+  ///
+  /// Default is false.
+  final bool? lockCameraToBuilding;
+
   /// The [MapView] settings. Required fields are your Situm user and API key,
   /// but also a buildingIdentifier or remoteIdentifier.
-  MapViewConfiguration({
-    this.situmUser,
-    required this.situmApiKey,
-    this.buildingIdentifier,
-    this.remoteIdentifier,
-    String? viewerDomain,
-    this.apiDomain = "dashboard.situm.com",
-    this.directionality = TextDirection.ltr,
-    this.enableDebugging = false,
-  }) {
+  MapViewConfiguration(
+      {this.situmUser,
+      required this.situmApiKey,
+      this.buildingIdentifier,
+      this.remoteIdentifier,
+      String? viewerDomain,
+      this.apiDomain = "dashboard.situm.com",
+      this.directionality = TextDirection.ltr,
+      this.enableDebugging = false,
+      this.lockCameraToBuilding}) {
```

Comment:
Si metes unha coma despois de this.lockCameraToBuilding e volves a formatear o texto, xa o pon como estaba antes.